sphinx_rtd_theme/layout.html

```diff
@@ -188,20 +188,27 @@
 
   {% if not embedded %}
 
-    <script type="text/javascript">
-        var DOCUMENTATION_OPTIONS = {
-            URL_ROOT:'{{ url_root }}',
-            VERSION:'{{ release|e }}',
-            LANGUAGE:'{{ language }}',
-            COLLAPSE_INDEX:false,
-            FILE_SUFFIX:'{{ '' if no_search_suffix else file_suffix }}',
-            HAS_SOURCE:  {{ has_source|lower }},
-            SOURCELINK_SUFFIX: '{{ sourcelink_suffix }}'
-        };
-    </script>
-    {%- for scriptfile in script_files %}
-      <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
-    {%- endfor %}
+    {% if sphinx_version >= "1.8.0" %}
```

Comment:
I think comparing versions as strings is a bad idea. For example, if Sphinx 1.10.0 is released, that version would be considered lower than 1.8.0 with such comparison.

Reply:
Parsing the version number would be better, but for simplicity sake this solution is OK provided that Sphinx will not release a 1.10, but will go to 2.x. (that's what I gather from http://www.sphinx-doc.org/en/master/changes.html)

@tk0miya will Sphinx go from 1.9 to 2.0, or might there be 1.10, 1.11 etc?

Reply:
Ok, the next version will probably be really 2.0 so in this particular case it's fine. In general it's still a bad idea :)

Reply:
Sorry for late. We'll move to 2.0 on next release, not 1.10.

>I think comparing versions as strings is a bad idea. 

Agreed. But there are no tuple-formed versions inside template.
It would be better if sphinx provides it in future.